src/Microsoft.ML.AutoML/EstimatorExtensions/EstimatorExtensions.cs

```diff
@@ -256,17 +257,45 @@ public IEstimator<ITransformer> CreateInstance(MLContext context, PipelineNode p
             return CreateInstance(context, pipelineNode.InColumns[0], pipelineNode.OutColumns[0]);
         }
 
-        public static SuggestedTransform CreateSuggestedTransform(MLContext context, string inColumn, string outColumn)
+        public static SuggestedTransform CreateSuggestedTransform(MLContext context, string inColumn, string outColumn, bool sort = false)
         {
             var pipelineNode = new PipelineNode(EstimatorName.ValueToKeyMapping.ToString(),
                 PipelineNodeType.Transform, inColumn, outColumn);
+            var estimator = CreateInstance(context, inColumn, outColumn, sort);
+            return new SuggestedTransform(pipelineNode, estimator);
+        }
+
+        private static IEstimator<ITransformer> CreateInstance(MLContext context, string inColumn, string outColumn, bool sort = false)
+        {
+            if(sort)
+                return context.Transforms.Conversion.MapValueToKey(outColumn, inColumn, keyOrdinality:ValueToKeyMappingEstimator.KeyOrdinality.ByValue);
+
+            return context.Transforms.Conversion.MapValueToKey(outColumn, inColumn);
+        }
+    }
+
+    internal class ImageLoadingExtension : IEstimatorExtension
+    {
+        public static string ImageFolder { private get; set; }
```

Comment:
Where do we Initialize `ImageFolder`

Or the image dataset has to be full_path?